src/Patient_Dashboard/UploadDocumentPage/FileTypeSection.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { RadioButtonGroup, RadioButton } from "carbon-components-react";
+
+const FileTypeSection = ({ selectedFileType, setSelectedFileType }) => {
+  return (
+    <div className="file-type-section">
+      <p className="file-type-heading">
+        File type <span className="important">*</span>
+      </p>
+      <div className="radio-buttons-section">
+        <RadioButtonGroup
+          name="fileType"
+          orientation="vertical"
+          valueSelected={selectedFileType}
+          onChange={(value) => setSelectedFileType(value)}
+        >
+          <RadioButton
+            value="Prescription"
+            labelText="Prescription"
+            className="radio-button"
+          />
+          <RadioButton
+            value="Referral Documents"
+            labelText="Referral Documents"
```

Comment:
These options come from Configuration. I think it is OK to hardcode in this PR. In future JIRA tickets we will have to make this come from configuration.

Reply:
I will raise the PR for this differently.